modules/server/templates/scripts/forseti-server/initialize_forseti_services.sh.tpl

```diff
@@ -39,7 +39,12 @@ FORSETI_COMMAND+=" --forseti_db $SQL_SERVER_LOCAL_ADDRESS/${cloudsql_db_name}?ch
 FORSETI_COMMAND+=" --config_file_path ${forseti_server_conf_path}"
 FORSETI_COMMAND+=" --services $FORSETI_SERVICES"
 
-CONFIG_VALIDATOR_COMMAND="${forseti_home}/external-dependencies/config-validator/ConfigValidatorRPCServer"
+CONFIG_VALIDATOR_COMMAND="$(which docker) run -d"
+CONFIG_VALIDATOR_COMMAND+=" --log-driver=gcplogs"
+CONFIG_VALIDATOR_COMMAND+=" --log-opt gcp-log-cmd=true"
+CONFIG_VALIDATOR_COMMAND+=" --log-opt labels=git-sync"
```

Comment:
The log labels for CV should be different than for git-sync, can we change the label to config-validator?

Reply:
Done